test/controllers/auth.controller.spec.js

```diff
@@ -85,7 +85,7 @@ describe('User authentication requests', () => {
         req.body = {
           email: user.email
         }
-        next = args => done(args)
+        let next = args => done(args)
```

Comment:
Every test should have its own context variables so they could be run independently. We could've used the `var` keyword, but no global variables should be used.